fractal_server/app/api/v1/project.py

```diff
@@ -320,6 +320,13 @@ async def apply_workflow(
         input_dataset_id=input_dataset_id,
         output_dataset_id=output_dataset_id,
         workflow_id=workflow_id,
+        workflow_dump=dict(
+            workflow.dict(exclude={"task_list"}),
+            task_list=[
+                dict(wf_task.task.dict(), task=wf_task.dict())
```

Comment:
Same as above: what about excluding `task`? Maybe it's a bit pedantic, but it's to make it clear that it won't be duplicated.

Reply:
:+1: